enhancements/SEP3-ambient-mode.md

```diff
@@ -0,0 +1,93 @@
+|Status                                             | Authors      | Created    | 
+|---------------------------------------------------|--------------|------------|
+|Proposed | @yxun         | 2024-11-19 |
+
+# Istio ambient mode
+
+## Overview
+
+Upstream Istio implements the ambient mode by splitting its functionality into two distinct layers. At the base, a per-node Layer 4(L4) ztunnel secure overlay handles routing and zero trust security for traffic. Above that, users can enable L7 waypoint proxies to get access the full range of Istio features.
+
+The upstream Istio ambient mode can be installed and upgraded from istioctl with profile ambient. This is implemented by the upstream in-cluster IstioOperator resource APIs.
+
+## Goals
+
+- Provide a way for installing the ambient mode component profile and charts in Sail Operator. This includes the ztunnel component chart and binary, Istio CNI changes and related meshConfig in pilot.
+- Provide a way for managing the ambient mode components life cycle. The implementation should enable in-place upgrades.
+
+## Non-goals
+
+- Uninstallation of only the ambient mode components. Currently, to cleanup the environment, need to compeletely uninstall Istio from a cluster.
+- Istio Sidecar mode migration to ambient mode.
```

Comment:
What about the following sentence?

Migration of Istiod control plane and dataplane components between sidecar mode and Ambient dataplane mode (and vice versa)